### ISSUE TITLE
LibLLVM::VERSION for gitted versions

### ISSUE THREAD

**Issue by HOMODELUNA** [crystal-lang/crystal#13696]
## Bug Report
```
In E:\lang\Crystal\src\llvm\lib_llvm.cr:32:35

 32 | IS_LT_90 = {{compare_versions(LibLLVM::VERSION, "9.0.0") < 0}}
                                    ^---------------
Error: Not a semantic version: "17.0.0git"
```
I pulled down LLVM from its git repository, and compiled it myself, so `llvm-config` emits a versionname suffixed with "git". To get a semantic version,the "git" suffix should be trimmed.
## Improvement
```crystal
# E:\lang\Crystal\src\llvm\lib_llvm.cr:13
VERSION = {{`#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify.gsub(/git/,"")}}
#                                                                ↑ `gsub` did this job
```
## Information
- crystal v1.8.2 (on Windows)
- llvm v17.0.0git

**Comment by straight-shoota** [comment]
Looks good. Would you mind creating a PR with the proposed fix?

**Comment by HOMODELUNA** [comment]
As posted in [PR13699](https://github.com/crystal-lang/crystal/pull/13699)

It's my first time to contribute to a real project, please report to me if I did something wrong.

**Comment by straight-shoota** [comment]
Thank you, looks great!
(it is a relatively simple fix afterall😄)

**Comment by HertzDevil** [comment]
Resolved by #13699
